lib/reporter/process.rb

```diff
@@ -47,6 +47,16 @@ def rss
 
     def collect_scheduler_stats(metric)
       metric.deferred_jobs_queued = Scheduler::Defer.length
+
+      if Discourse.respond_to?(:job_exception_stats)
+        failures = {}
+        Discourse.job_exception_stats.each do |klass, count|
+          failures[{ "job" => klass.to_s }] = count
+        end
+        metric.scheduled_job_failures = failures
+      else
+        metric.scheduled_job_failures = {}
+      end
```

Comment:
```suggestion
      metric.scheduled_job_failures = {}

      if Discourse.respond_to?(:job_exception_stats)
        Discourse.job_exception_stats.each do |klass, count|
          metric.scheduled_job_failures[{ "job" => klass.to_s }] = count
        end
      end
```

I think this will work too

Reply:
yeak way simpler